javascript-data-model/order-history.js

```diff
@@ -0,0 +1,80 @@
+
+var orderHistory = [
+  {
+    orderPlaced: 'Auguest 4, 2020',
+    total: 34.00,
+    shipTo: 'JS Masher',
+    orderNum: '114-3941689-87722232',
+    deliveredDate: 'Delievered Aug,8 2020',
+    deliveryMethod: ' ',
+    item: 'Javascript for Impatient programmers',
+    author: 'Rauschmayer Dr Axel',
+    rtrnWdindow: 'Sep 7,2020',
+    price: 31.00,
+    buyItAgain: true,
+    viewItem: true,
+    productReview: true,
+    orderRetails: true,
+    invocie: true,
+    productSupport: false
+  },
+
+  {
+    orderPlaced: 'July 19,2020',
+    total: 44.53,
+    shipTo: 'JS Masher',
+    orderNum: '133-9984268-1280257',
+    deliveredDate: 'Delievered Jul 20, 2020',
+    deliveryMethod: 'Your package was delivered It was handed directly handed to a resident',
+    item: 'The Timeless Way of Building',
+    author: 'Alexander Christopher',
+    rtrnWindow: 'Aug 19,2020',
+    price: 41.33,
+    buyItAgain: true,
+    viewItem: true,
+    productReview: true,
+    orderDetails: true,
+    invocie: true,
+    productSupport: false
+  },
+
+  {
+    orderPlaced: 'July 4,2020',
+    total: 17.22,
+    shipTo: 'JS Masher',
+    orderNum: '114-2875557-9059409',
+    deliveredDate: 'Delievered Jul 7, 2020',
+    deliveryMethod: 'Your package was delivered.It was handed directly to a resident',
+    item: 'Gamecube Contorller Adapter.Super Smash Bros Switch Gamecube Adapter fow Wii U Pc.Support Turbo and Vibration features.No Driver and No Lag-Gamecube Adapter',
+    author: ' ',
+    rtrnWindow: 'Aug 5,2020',
+    price: 15.98,
+    buyItAgain: true,
+    viewYourItem: true,
+    productReview: true,
+    orderDetails: true,
+    invocie: true,
+    productSupprt: false
+  },
+
+  {
+    orderPlaced: 'July 3,2020',
+    total: 138.93,
+    shipTo: 'JS Masher',
+    orderNum: '113-2883177-2648248',
+    deliveredDate: 'Delievered Jul 5, 2020',
+    deliveryMethod: ' ',
+    item: ['Gamecube Contorller-Super Smash Bros.edition (Nintendo Switch) Pc Support Turbo and Vibration features No Driver and No Lag -Gamecube Adapter', 'the Art of Sql'],
+    author: [' ', 'Faroult Stephane'],
+    returnWindow: ['Aug 4,2020', 'Aug 4,2020'],
+    price: [94.55, 33.99],
+    buyItAgain: [true, true],
+    viewYourItem: [true, true],
+    productReview: [true, true],
+    orderDetails: [true, true],
```

Comment:
This object has a bunch of arrays, but all the other objects have a different format. That is not consistent, and it will lead to bugs. Make sure each object is formatted in the same way. Also, rather than making a bunch of individual arrays, I suggest one array with an object for each item in the order.